aws-rds-dbinstance/src/main/java/software/amazon/rds/dbinstance/util/ResourceModelHelper.java

```diff
@@ -11,7 +11,7 @@ public final class ResourceModelHelper {
     private final static String STORAGE_TYPE_IO1 = "io1";
 
     public static boolean shouldUpdateAfterCreate(final ResourceModel model) {
-        return (isReadReplica(model) || isRestoreFromSnapshot(model) || isCertificateAuthorityApplied(model)) &&
+        return (isReadReplica(model) || isRestoreFromSnapshot(model) || isCertificateAuthorityApplied(model) || isRestoreToPointInTime(model)) &&
```

Comment:
What exact attributes are we aiming to set with a modify after restore to point in time?

Reply:
Everything that is missing from this list is:
1. AllocatedStorage
2. ca-certificate-identifier
3. engine-version
4. master-user-password
5. preferred-backup-window
6. preferred-maintenance-window
7. backup-retention-period


So presumably all of these and any additional properties that have not been set from the restore.

Reply:
Ack. Thanks.